services/user.js

```diff
@@ -428,12 +429,13 @@ class UserService {
             throw new Error(SUBMODULE_ERROR.USER_NOT_FOUND);
         }
         const updatedUser = {};
+        // TODO
         const isCurator = updatedUser?.role === USER.ROLES.CURATOR || user[0]?.role === USER.ROLES.CURATOR || params?.role === USER.ROLES.CURATOR;
```

Comment:
Seems isCurator is only used later in line 452 to set organization to null, I don't think it's still needed. So I suggest we remove this line and line 452-454

Reply:
I've updated the change.